deploy/operator/capi/deploy_capi_cluster.sh

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+__root="$(realpath ${__dir}/../../..)"
+source ${__dir}/../common.sh
+source ${__dir}/../utils.sh
+source ${__dir}/none_platform_utils.sh
```

Comment:
```
./deploy_capi_cluster.sh: line 7: /home/assisted-service/deploy/operator/capi/none_platform_utils.sh: No such file or directory
```
in https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_assisted-service/3690/pull-ci-openshift-assisted-service-master-e2e-metal-assisted-operator-capi/1517490862534168576